components/cache/adapters/redis_adapter.rst

```diff
@@ -188,6 +188,20 @@ Available Options
     When using the `Predis`_ library some additional Predis-specific options are available.
     Reference the `Predis Connection Parameters`_ documentation for more information.
 
+.. _redis-tag-aware:
+
+Working with tags
+-----------------
+
+In order to use tags-based invalidation, you can wrap your adapter in :class:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapter`. But when Redis is used as backend, it's often more interesting to use the dedicated adapter :class:`Symfony\\Component\\Cache\\Adapter\\RedisTagAwareAdapter`. Since tag invalidation logic is implemented in Redis itself, this adapter offers better performance when using tags-based invalidation::
+
+    use Symfony\Component\Cache\Adapter\RedisAdapter;
+    use Symfony\Component\Cache\Adapter\TagAwareAdapter;
+
+    $client = RedisAdapter::createConnection('redis://localhost');
+    $cache = new TagAwareAdapter($client);
```

Comment:
Shouldn't we use `RedisTagAwareAdpter` in this example if we use Redis here? Otherwise I would reccommend to NOT use Redis in this example and use another backen.

WDYT?

Reply:
Indeed thank you :) It was a typo, I actually wanted to use RedisTagAwareAdapter in my example.